web/src/beta/features/PluginPlayground/Code/utils.ts

```
@@ -0,0 +1,30 @@
+const REEARTH_HTML_INJECTION_PATTERN =
+  /reearth\.(ui|modal|popup)\.show\(\s*(['"`])([^]*?)\2\s*\);/;
```

Biome

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)